index.css

```diff
@@ -1,567 +1,1655 @@
-html body {
-  margin: 0;
-  padding: 0;
-}
+/* This is the mobile version of the Portfolio */
 
-.main-container {
-  display: flex;
-  flex-direction: column;
-}
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] [optional] @Shuhad786 it's not good practice having media queries for mobile-first design, as you are required to use the queries for desktop or tablet versions.